# name: try
# key: try
# --
try {
    $1
} catch (Exception e) {
    e.printStackTrace();
}$0